### PR TITLE
Fix to-have-length when there are no argument properties

### PR DESCRIPTION
**Pull request by SimenB**
@Ionut-Milas

### CODE REVIEW

File: rules/prefer_to_have_length.js [hunk 1]
```diff
@@ -17,7 +17,11 @@ module.exports = context => {
         const argumentObject = node.arguments[0].object;
         const argumentProperty = node.arguments[0].property;
 
-        if (propertyName === 'toBe' && argumentProperty.name === 'length') {
+        if (
+          argumentProperty &&
+          propertyName === 'toBe' &&
```
Comment (thymikee) [inline rules/prefer_to_have_length.js]: just a nit, can we switch these two lines?

Comment (SimenB) [reply rules/prefer_to_have_length.js]: what do you mean?

Comment (SimenB) [reply rules/prefer_to_have_length.js]: Oh, gotcha

Comment (thymikee) [reply rules/prefer_to_have_length.js]: Sorry for being indirect :D

**Comment by SimenB** [comment]
The rule now runs successfully on the jest codebase as well

**Pull request merged**